### ISSUE TITLE
Can  winappdriver be used to automate ms access applications? 

### ISSUE THREAD

**Issue by suvendu3s** [microsoft/WinAppDriver#1280]
I have been trying to use winappdriver to automate ms access applications. But I'm notable to interact with they elements present inside the sandbox.

Please suggest.

**Comment by anunay1** [comment]
Is inspect able to find the elements

**Comment by suvendu3s** [comment]
> Is inspect able to find the elements inside the sandbox

Yes, it is but the script is not able to access it. Only the name attribute is available for the elements. I have tried creating xpaths also still no results .

**Comment by anunay1** [comment]
Can you share the console logs, also it would be good if you could share any screenshot of the application.

**Comment by DLightstone** [comment]
You are going to have to be a bit more specific as to what you which to test

I have never attempted to  useWinAppdriver to automate Access itself 
I have however used WinAppdriver to test an Win32 application that uses Access Automation and Word Automation as infrastructure.
